hrm-domain/hrm-core/permissions/actions.ts

```diff
@@ -36,8 +36,6 @@ export const actionsMaps = {
     ADD_DOCUMENT: 'addDocument',
     EDIT_DOCUMENT: 'editDocument',
     EDIT_CASE_OVERVIEW: 'editCaseOverview',
-    EDIT_CHILD_IS_AT_RISK: 'editChildIsAtRisk',
-    EDIT_FOLLOW_UP_DATE: 'editFollowUpDate',
```

Comment:
Just to remind me, we're removing these because they're (currently) always `true` when `EDIT_CASE_OVERVIEW` is `true`?

Reply:
Yes, we are merging the 3 old permissions: EDIT_CHILD_IS_AT_RISK, EDIT_FOLLOW_UP_DATE, EDIT_CASE_SUMMARY into a single EDIT_CASE_OVERVIEW permission that effectively sets all 3 old permissions with whatever it spoecifies